tensorboard/webapp/metrics/views/card_renderer/utils.ts

```diff
@@ -12,22 +12,25 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 ==============================================================================*/
+import {ExperimentAlias} from '../../../experiments/types';
 import {Run} from '../../../runs/store/runs_types';
 import {LinkedTime} from '../../types';
 import {PartialSeries, PartitionedSeries} from './scalar_card_types';
 
 export function getDisplayNameForRun(
   runId: string,
   run: Run | null,
-  experimentName: string | null | undefined
+  experimentAlias: ExperimentAlias | null | undefined
```

Comment:
AFAICT, scalar_card_container is another place that uses this utility. Do we need to change anything there for this change? I don't see that module in the changelist. Especially since we have set it to `null` in one of two places that this utility is being used, I suspect we can probably get rid of this utility in favor of common widget or something like that. WDYT?